components/i2c/htu31d/definition.json

```diff
@@ -0,0 +1,11 @@
+{
+    "displayName": "HTU31D",
+    "i2cAddresses": [ "0x40", "0x41" ],
+    "subcomponents": [
+    {
+      "displayName": "Temperature/Humidity Sensor",
```

Comment:
Please make the `displayName` less than 24 characters

Our CI reported this error here: https://github.com/adafruit/Wippersnapper_Components/runs/8004594245?check_suite_focus=true#step:4:27